src/data_source/storage/sql.rs

```diff
@@ -587,6 +587,8 @@ impl SqlStorage {
                 .await?;
         }
 
+        conn.close().await?;
```

Comment:
Doesn't this happen automatically when `conn` is dropped?

Reply:
I was trying the locking_mode to exclusive and db was stuck after migrations. closing the connection explicitly fixed it